### ISSUE TITLE
TypeError: Path must be a string. Received -1

### ISSUE THREAD

**Issue by cpunion** [timarney/react-app-rewired#21]
Configured by following README.md, met error:

```js
✗ yarn start
yarn start v0.19.1
$ react-app-rewired start
/Users/lijie/test -1
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received -1
    at assertPath (path.js:7:11)
    at Object.join (path.js:1211:7)
    at Object.<anonymous> (/Users/lijie/test/node_modules/react-app-rewired/config/paths.js:27:31)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
```

See https://github.com/timarney/react-app-rewired/blob/master/packages/react-app-rewired/config/paths.js#L17-L26, seems `custom_scripts` always -1, and then
`const scriptVersion = custom_scripts || 'react-scripts'` also -1.

My environment:
Node.js 7.4.0
react-scripts 0.9.0
react-app-rewired 0.6.4

**Comment by timarney** [comment]
@cpunion should be corrected in 0.6.5

**Comment by cpunion** [comment]
@timarney Thank you for quickly fix.
